apps/vr-tests/package.json

```diff
@@ -35,7 +35,7 @@
     "@types/react-dom": "16.8.4",
     "@types/react": "16.8.25",
     "@types/webpack-env": "1.15.1",
-    "awesome-typescript-loader": "^3.2.3",
+    "awesome-typescript-loader": "^5.2.1",
```

Comment:
TBD whether this upgrade is necessary/desirable and whether it necessitates other changes